src/components/EditImageModal/index.jsx

```diff
@@ -533,7 +643,42 @@ export default class EditImageModal extends React.Component {
     </React.Fragment>
   );
 
-  getModalInsertImageButton = () => (
+  getModalButtons = () => {
+    let buttons;
+    if (this.state.pageNumber === 1) {
+      buttons = this.getNextPageButton();
+    } else if (this.state.pageNumber === 2) {
+      buttons = this.getInsertImageButton();
+    }
+    return buttons;
+  }
+
+  getNextPageButton = () => (
+    <Button
+      label={
+        <WrappedMessage
+          message={messages.editImageModalNextPageButton}
+        />
+      }
+      buttonType="primary"
+      onClick={this.onNextPageButtonClick}
+    />
+  );
+
+  getPreviousPageButton = () => (
+    <Button
+      label={
+        <WrappedMessage
+          message={messages.editImageModalPreviousPageButton}
+        />
+      }
+      buttonType="link"
+      onClick={this.onPreviousPageButtonClick}
```

Comment:
I think you should bind this as well.

Reply:
Can do!

Reply:
I don't think I see this being bound? Am I missing it somewhere?